CODEOWNERS

```diff
@@ -1,5 +1,168 @@
 # CODEOWNERS for autoreview assigning in github
 
 # Get all docs reviewed
-doc/	@dbkinder
-*.rst	@dbkinder
+
+*                                        kumar.gala@linaro.org anas.nashif@intel.com
+*.rst                                    @dbkinder
+.known-issues/*                          inaky.perez-gonzalez@intel.com anas.nashif@intel.com
+MAINTAINERS                              inaky.perez-gonzalez@intel.com anas.nashif@intel.com
+arch/arc/*                               cjordan@synopsys.com Ruud.Derwig@synopsys.com
+arch/arc/core/*                          andrew.p.boie@intel.com
+arch/arm/*                               maureen.helm@nxp.com kumar.gala@linaro.org
+arch/arm/core/*                          andrew.p.boie@intel.com
```

Comment:
You should verify this, but as I read the (limited) documentation about CODEOWNERS, the last pattern match takes precedence, so if a file is updated in arch/arc/core/* only Andrew will be  noted as the owner and not Maureen and Kumar (as owners of the parent folder).  Is that the behavior you're intending?